specs/DevSpecs/telemetry.md

```diff
@@ -0,0 +1,187 @@
+# Input.Rating Telemetry Design Specification 
+ 
+## Abstract 
+This design specification covers telemetry integration into the `Input.Rating` control, which will be included in a future release of Adaptive Cards. Telemetry integration is currently slated only for the JavaScript platform; however, this developer specification will be general enough to cover all platforms supported by Adaptive Cards. This telemetry design will focus on sending telemetry via the 1DS platform, but the design is flexible enough to allow for possible integration of other telemetry platforms in the future. See more at the GitHub issue [3058](https://github.com/microsoft/AdaptiveCards/issues/3058).
+ 
+## 1. Feature Dependencies 
+### 1.1 Dependencies on other features 
+Because telemetry will be baked into the `Input.Rating` control as a result of the implementation of this design, this feature is inherently dependent on the `Input.Rating` control. Externally, this feature is dependent on the functionality of the 1DS SDK. 
```

Comment:
I wouldn't call Input rating control a dependency perse here. 1DS on the other hand is.